global/utils/validateUser.ts

```diff
@@ -0,0 +1,17 @@
+import { WalletUser } from '../types';
+
+// not a true validation, will just ensure all expected properties are present
+const validateUser: (user: any) => WalletUser = (user) => {
+  const output: WalletUser = {
+    name: '',
+    email: '',
+    id: '',
+  };
+
+  return {
+    ...output,
+    ...user,
+  } as WalletUser;
+};
```

Comment:
Fine for this case, since we trust our backend structure, but as discussed this has some potential issues:
- add more properties to output than we expect
- Could replace values with non-string type values.
Don't need to change anything, just commenting for the record.